bundles/SyliusProductBundle/installation.rst

```diff
@@ -60,28 +65,62 @@ Put this configuration inside your ``app/config/config.yml``.
 
 .. code-block:: yaml
 
+    sylius_archetype:
+        classes:
+            product:
+                subject: Sylius\Component\Product\Model\Product
+                attribute: Sylius\Component\Product\Model\Attribute
+                option: Sylius\Component\Product\Model\Option
+                archetype:
+                    model: Sylius\Component\Product\Model\Archetype
+                    repository: Sylius\Bundle\ResourceBundle\Doctrine\ORM\TranslatableEntityRepository
+                    translatable:
+                        targetEntity: Sylius\Component\Product\Model\ArchetypeTranslation
+                archetype_translation:
+                    model: Sylius\Component\Product\Model\ArchetypeTranslation
+
+    sylius_attribute:
+        driver: doctrine/orm
+
     sylius_product:
-        driver: doctrine/orm # Configure the doctrine orm driver used in the documentation.
+        driver: doctrine/orm
+
+    sylius_locale:
+        driver: doctrine/orm
+
+    sylius_translation:
+        default_mapping:
+            translatable:
+                field: translations
+                currentLocale: currentLocale
+                fallbackLocale: fallbackLocale
+            translation:
+                field: translatable
+                locale: locale
+
+    sylius_variation:
+        driver: doctrine/orm
+
 
-And configure doctrine extensions which are used by the bundle.
+Then configure the locale to be used by Sylius services. If you are working with Symfony 2.6,
+you can put this configuration inside your ``app/config/services.yml`` or in versions prior
+to that, in ``app/config/config.yml``
 
 .. code-block:: yaml
 
-    stof_doctrine_extensions:
-        orm:
-            default:
-                sluggable: true
-                timestampable: true
+    parameters:
+        sylius.locale: "%locale%"
 
-Routing configuration
----------------------
 
-Add the following to your ``app/config/routing.yml``.
+Then configure Doctrine extensions which are used by the bundle.
 
 .. code-block:: yaml
 
-    sylius_product:
```

Comment:
Because there is no `@SyliusProductBundle/Resources/config/routing.yml` resource - this was removed as routing for the product is now managed by the CMF router. We'll need to document the changes made by @lsmith77 in https://github.com/Sylius/Sylius/commit/4488efd0cff364047f307711e2de513500d271ac (unfortunately the CMF router is still a bit of a black box to me).